gltf/gltfpack.cpp

```diff
@@ -676,6 +722,8 @@ static void process(cgltf_data* data, const char* input_path, const char* output
 	writeArray(json, "meshes", json_meshes);
 	writeArray(json, "skins", json_skins);
 	writeArray(json, "animations", json_animations);
+	comma(json);
+	append(json, json_extensions);
```

Comment:
This will always emit "extensions" and also causes a conflict with top-level KHR_lights_punctual extension since there's now going to be two extension blocks for files that use that.

Instead what should happen is:
- `json_extensions` should be initially empty
- `json_lights` should be appended to it, if necessary
- VRM data should be appended to it, if necessary
- if `json_extensions` isn't empty when forming the final JSON blob, we should write out `"extensions":{` followed by json_extensions contents followed by `}`